exchanges/stats/stats_types.go

```diff
@@ -0,0 +1,29 @@
+package stats
+
+import (
+	"sync"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+)
+
+var (
+	// items holds stat items
+	items     []Item
+	statMutex sync.RWMutex
```

Comment:
This is already so out of scope: The entry point is in engine/sync_manager.go when `PrintTickerSummary` is called. The data doesn't get changed once its set, this is going to be smashed by a main lock call with the append function. It's probably more efficient to use a sync.Mutex, or have a map to and localise a normal sync.Mutex at the Item struct. Or RM the package (My personal preference option) completely because we can build and find ticker data from the ticker package. We are just duplicating data, insertion and updating at a less efficient way.

Reply:
Hey I tried not to rock the boat until you expanded the scope with your damn races 😄 I agree with removal, just didn't want to touch another system

Reply:
![image](https://github.com/thrasher-corp/gocryptotrader/assets/9391715/a4bb1880-0d30-4d02-a8d2-1a6d9970b2b2)

Reply:
I'm just going to change it to a normal mutex, anything further requires a lot of work that isn't fixing the Kraken panic. Removal of `stats` package has other consequences for exported `engine` functions that I don't want to interfere with.